modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -4,77 +4,203 @@ import {Meta} from '@storybook/addon-docs';
 
 # Canvas Kit 8.0 Upgrade Guide
 
-This guide contains breaking changes in Canvas Kit v8. Please
+This guide contains an overview of the changes in Canvas Kit v8. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
 - [Codemod](#codemod)
-- [Components](#components)
-  - [Component Deprecations](#component-deprecations)
+- [General Updates](#general-updates)
+  - [IE11 No Longer Supported](#ie11-no-longer-supported)
+  - [Removal of Default Exports](#removal-of-default-exports)
+  - [Responsive Styles](#responsive-styles)
+  - [Refactor Style Props](#refactor-style-props)
+  - [Testing](#testing)
+- [New Components](#new-components)
+  - [Grid](#grid)
+  - [Text](#text)
+  - [Status Indicator (Preview)](#status-indicator-preview)
+- [Component Deprecations](#component-deprecations)
+  - [Drawer](#drawer)
+  - [Menu (Preview)](#menu-preview)
+  - [Layout and Column](#layout-and-column)
+- [Component Updates](#component-updates)
   - [Box](#box)
-  - [Menu (Main)](#menu)
   - [Breadcrumbs](#breadcrumbs)
-- [General Changes](#general-changes) -[Remove Default Imports](#remove-default-imports)
+  - [Menu (Main)](#menu-main)
+  - [Modal](#modal)
+  - [Segmented Control](#segmented-control)
 
 ## Codemod
 
 Please use our [codemod package](https://github.com/Workday/canvas-kit/tree/master/modules/codemod)
-to automatically update your code to work with many of the breaking changes as you upgrade from
-Canvas Kit v6 to v7:
+to automatically update your code to work with many of the breaking changes in v8.
```

Comment:
should we leave out `many` ? maybe just say `with the breaking changes introduced in v8? Makes it seems like v8 is an odd number release. Thoughts?

Reply:
I've changed `many` to `most` to minimize the possibility that someone would interpret v8 as having many breaking changes.